tests/e2e/e2e_test.go

```diff
@@ -85,24 +85,24 @@ var _ = Describe("odoe2e", func() {
 		})
 	})
 
-	Context("odo utils config", func() {
+	Context("odo config", func() {
 		It("should get blank for updatenotification by default globally as its not set", func() {
-			configOutput := runCmdShouldPass("odo utils config view --global")
-			Expect(configOutput).To(ContainSubstring(config.UpdateNotificationSetting))
-			Expect(configOutput).To(ContainSubstring(config.NamePrefixSetting))
-			Expect(configOutput).To(ContainSubstring(config.TimeoutSetting))
+			configOutput := runCmdShouldPass("odo preference view")
+			Expect(configOutput).To(ContainSubstring(preference.UpdateNotificationSetting))
+			Expect(configOutput).To(ContainSubstring(preference.NamePrefixSetting))
+			Expect(configOutput).To(ContainSubstring(preference.TimeoutSetting))
 		})
 		It("should be checking to see if timeout is shown as blank globally as its not set", func() {
-			configOutput := runCmdShouldPass("odo utils config view --global|grep Timeout")
+			configOutput := runCmdShouldPass("odo preference view |grep Timeout")
 			Expect(configOutput).To(ContainSubstring(fmt.Sprintf("Timeout")))
 		})
 		It("should be checking to see if global config values are the same as the configured ones", func() {
-			runCmdShouldPass("odo utils config set --global updatenotification false")
-			runCmdShouldPass("odo utils config set --global timeout 5")
-			configOutput := runCmdShouldPass("odo utils config view --global |grep UpdateNotification")
+			runCmdShouldPass("odo preference set updatenotification false")
+			runCmdShouldPass("odo preference set timeout 5")
+			configOutput := runCmdShouldPass("odo preference view |grep UpdateNotification")
```

Comment:
To avoid terminal specific command ```grep```. Use function like

```
func findLocalConfigValueOfGivenKey(key string) string {

	stdOut, _, _ := cmdRunner("odo config view")
	re := regexp.MustCompile(key + `.+`)
	odoConfigKeyValue := re.FindString(stdOut)
	if odoConfigKeyValue == "" {
		return fmt.Sprintf("%s not found", key)
	}
	trimKeyValue := strings.TrimSpace(odoConfigKeyValue)
	if strings.Compare(key, trimKeyValue) != 0 {
		return strings.TrimSpace(strings.SplitN(trimKeyValue, " ", 2)[1])
	}
	return ""
}

func findGlobalConfigValueOfGivenKey(key string) string {
	stdOut, _, _ := cmdRunner("odo preference view")
	re := regexp.MustCompile(key + `.+`)
	odoConfigKeyValue := re.FindString(stdOut)
	if odoConfigKeyValue == "" {
		return fmt.Sprintf("%s not found", key)
	}
	trimKeyValue := strings.TrimSpace(odoConfigKeyValue)
	if strings.Compare(key, trimKeyValue) != 0 {
		return strings.TrimSpace(strings.SplitN(trimKeyValue, " ", 2)[1])
	}
	return ""
}
```

Reply:
Add this function in utils.go file in testsuite and call it in the test file

Reply:
This should be probably handled in different PR.

Reply:
No issue, Anyway i am handling it in a separate pr #1324

Reply:
resolving this